grout/models.py

```diff
@@ -52,43 +75,87 @@ def validate_schema(self, schema):
         jsonschema.Draft4Validator.check_schema(schema)
 
 
-class Record(GroutModel):
-    """Spatiotemporal records -- e.g. Loch Ness Monster sightings, crime events, etc."""
+class AbstractFlexibleRecord(GroutModel):
+    """
+    Base class for flexible records.
+    """
+    schema = models.ForeignKey('RecordSchema', on_delete=models.CASCADE)
+    data = JSONField()
+    archived = models.BooleanField(default=False)
+
+    class Meta(object):
+        abstract = True
+
+
+class DateTimeRange(models.Model):
+    """
+    Save a flexible record with date and time attributes.
+    """
     occurred_from = models.DateTimeField()
     occurred_to = models.DateTimeField()
 
-    geom = models.PointField(srid=settings.GROUT['SRID'])
-    location_text = models.CharField(max_length=200, null=True, blank=True)
+    class Meta(object):
+        abstract = True
 
-    schema = models.ForeignKey('RecordSchema', on_delete=models.CASCADE)
-    data = JSONField()
 
-    archived = models.BooleanField(default=False)
+class FlexibleRecord(AbstractFlexibleRecord):
```

Comment:
`FlexibleRecord` and `TemporalFlexibleRecord` represent the non-geospatial types of Records introduced in this PR. While `Record` and `TemporalRecord` would probably be cleaner names, it seems important to me that we preserve the current `Record` class for backwards-compatibility.